src/main/java/com/kainos/ea/dao/RolesDao.java

```diff
@@ -30,4 +33,28 @@ public List<JobRole> getAllRoles(Connection c) throws SQLException {
         }
 
     }
+    public List<JobSpecification> getAllSpecification(Connection c, int role_id) throws SQLException, RoleNotExistException {
```

Comment:
this shouldn't return a list as it will only return 1 specification

Reply:
done